blockchain/reactor.go

```diff
@@ -228,22 +228,23 @@ func (bcR *BlockchainReactor) poolRoutine() {
 
 	didProcessCh := make(chan struct{}, 1)
 
-FOR_LOOP:
-	for {
-		select {
-		case request := <-bcR.requestsCh:
+	go func() {
```

Comment:
Not sure how extracting `requestsCh` into its own goroutine solves anything... but I don't understand the issue well enough

Reply:
The topic of the issue is that :  write a  BlockRequest int requestsCh channel will create an timer at the same time that stop the peer 15s later if no block have been received . But pop a BlockRequest from requestsCh  and send it out may delay more than 15s later.  So that the peer will be stopped for error("send nothing to us").
 Extracting requestsCh into its own goroutine can make sure that every BlockRequest been handled timely.